stable/yugaware/values.yaml

```diff
@@ -29,6 +29,13 @@ tls:
   key: "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBd0prMFNubm9aUWFtYVptKzFVSXY3UWIwS2pYWXlRMHJKSUhqbGxNN3pYRWFBaC9CCmpWNnUzSHVNbGtOVlJ6RUVGeEtGUm5nSlpXTVlPcy9sWDU1QmtBS0RocXNmaDl0ZVA4ZFVQU1RTVTN6bWRVc0cKU3AySDBqSm1YbkVjbzBac3BiVXJhaGVxaXFJRTJnWDBCWm9iV29YZEdzUTJ1cmJQbHIwaVVKdTFWenhIZGhNLwpnTnFQWEN5bE5YdENrbVI2NHl6SGtEaG9taUUwMkkzME1wZThEcXM5eHd5Sndxek9ydmZwNHlBNFJIZ2toNXJaCi8rUStzRVA2QjJXMEdBdzZzZDNTYytOTytYVUZoVnlFRDhqZkNsTndPVHZodi9keE5IWHZ4WHd0RWNjZTd1TVAKQUxSc0hYdUVtZE5DeTdqb2JvMm5RYU5vNkFDVzRTazlFYlVlcHdJREFRQUJBb0lCQUJDbDdaZzJlM2tvSHdCbgowdEFhWFl3dW1ySi9YeExaK1kyNVVKUHc4KzFwUm1kQ21YZC81THVZVG1tNnZrZGVIRTFtUTNENDlVNzNSdVBtCmpwdzZVTTZDK1IyTjh6YktKcFo2dzNSWmwzaXlCalhEVzNMZHRSemRVRzMvZHd0Q0RpZ3d4ZVR3ME1RVkJ6aFYKWldlS2o2eStmNE5OekxGTkZocWdsTFBGRWVMTlk3WUgvYUdHdTZwaVpERkFMT0liMlFVUUVvVVRkUEJabDMxSwpKL0Y0cHE1RkV6ekQ0aHMwYXJzSlIxeUd4L0NCRWxqQWRSNWE4Y3FHdGhxYjFMdXJTMVcxT0dNUGkzNFdDblJKCnYwTC9zSU9ORCt5cjdjd3ZQaFFzcmI3Tk9NbGxRSDYwUGdyQ0JyOUVwalFiRm1ZN3NCTS9jYUZTWHVDTmZnTzMKT3hTQjF0RUNnWUVBNHpSMTh2aXl3Ykt5aE5xeEhkNzJ1UkxHS2k4SWZhVmhsNGxFeGdsR0pLNHQ0KzYreXhObAp2OFBkdGg3elJjL1pyQUlBUEtBRnZxa1RnVzkrc2llL3JPcE9PUzA2UnVIWVpJMjdpS3ZiL054N29YTm9TM0Y2CitXRlNJSVJqRFo0eHc2SU1iNUJFS0hOVGtKU1pBMXBkaXhYeFgzeDVTM2hCWTlmekl2NGdVVDhDZ1lFQTJRSDAKbnU5REZ0MDNmWmpvcjRMUkZCQS84NEV1dmdBL2RxMkZ4TzVvMjIrdk43Qk9LRW9IWTRuTzhYdFN2L2MzaVFZdApmTUdGZUR2eXdqRnFPU05SS2JxenMza3ZmY2gxNEhHdFltT29DVTF6Smd2VkF4SytWOFF4YWxMbmhlbjZsaTgwCnJKYmtWWUJXYnFHZHU1d2FPM1RyeGsvbHBMejFvSnliK0FQbmNKa0NnWUVBeHd3anRTeEtOc1hkL2VRMXZReWoKcTdvVjM4ajZzRkRkOGNzeGRDbVZ5UGJMZjErZXlQeUNiSUVrMExQR3ZzZDhWR1FnWHZ4MmEvVm1BRVIwZzVHdApLOGpOcE92Sm5XV29sVmRTRDBjNDRBc3RTR0Y1NG5aVm5QcTAzNFg3M1dkVkE2WUpMUWYvbmt5Q3ppeFE3UTRTCjI1SHF0NU91RS84VGttd1ROdm5aeURNQ2dZQlUxV0xmendFWEt4dktTY0NNL0VMbjZRSlhHeVUrdWJJVm52Z3cKYUZNdGxaZVYyYUowTlp4MTJTWEJXVWljS3ZUVzZFZFlBZjgvWlpBZFFsd1lRaW8wZWZ4T0tRM2J5a1hxbkFYLwo5SjE0SllMaVdMWEpJY2F2WU05QWxYQlU5WWMwdHZlTWtPQnFYR1Z0a2xpclF2Y3VUeUxzdjhjeWczd3ljWWEzCmlpOHI0UUtCZ0NaODZtcEZuUlN2aTdpTWw1MUdybDFQZER2YzZVMURRVkI5empTQlVzak1lamNTQXk4WmVJZlQKUDVoSGc3REFaUEhmUGpYLzZxUnpNQW8zWlFlZVBMdTNEYVgwbFBoekNhalF1bkpIbWZvYlA2QjFOUG45T056bQo0dE0wdEU1YTJVVmJuZW5haVZyaDRhSXlJTTFxVTdHT1ArcDNMNi9hNzVnSVQ2SVJRVlloCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg=="
 
+## yugaware pod Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+securityContext:
+  enabled: false
+  fsGroup: 1001
+  runAsUser: 1001
```

Comment:
@ramkumarvs will the yugabyte user always have UID of 1001?

Reply:
Code changes look good to me. Although, there is a further refinement that can be done by specifying `runAsUser: 'MustRunAsNonRoot'` based on the k8s pod-security-policy docs [here](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#users-and-groups). Looks like `MustRunAsNonRoot` need to be used in conjunction with `allowPrivilegeEscalation=false` attribute.

Reply:
@daniel-yb yes we explicitly set that in the RHEL docker image.